doc/environment-rtd.yml

```diff
@@ -15,6 +15,7 @@ dependencies:
   - gdal=3.3.0
   - pip=21.1.3
   - richdem=0.3.4
+  - pillow
```

Comment:
The idea in `doc/environment.yml` is to pin everything to a compatible version set. This is done to make the Conda solve step pass on the RTD runner. Otherwise, we could just use the main `environment.yml`.